Example Measure.json

```diff
@@ -0,0 +1 @@
+{"name":"Example Measure","remarks":[],"states":{"Initial":{"type":"Initial","direct_transition":"Initial"},"Terminal":{"type":"Terminal","direct_transition":"Terminal"},"Procedure_1":{"type":"Procedure","direct_transition":"Procedure","codes":[{"code":"","system":"","display":"","value_set":"http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113883.3.464.1003.108.12.1020"}]},"Encounter_2":{"type":"Encounter","direct_transition":"Encounter","codes":[{"code":"","system":"","display":"","value_set":"http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113883.3.464.1003.101.12.1001"}],"encounter_class":"encounter_class"},"Medication_3":{"type":"MedicationOrder","direct_transition":"Medication","codes":[{"code":"","system":"","display":"","value_set":"http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113883.3.464.1003.108.12.1020"}]}}}
```

Comment:
I think we can just delete this. Was this an example output from a run that you accidentally committed?